ui/src/styles/theme.css

```
@@ -0,0 +1,62 @@
+  /* Tungsten amber */
+  --accent: #e0a55a;
+  --accent-warm: #d97a3a;
+  --accent-dim: #8a6537;
+
+  /* Type */
+  --text: #e8e6df;
+  --text-muted: #8a8780;
+  --text-faint: #5d5b56;
+
+  /* Faces */
+  --display: "EB Garamond", "Iowan Old Style", Georgia, serif;
```

Stylelint (17.9.0)

[error] Expected "Georgia" to be "georgia" (value-keyword-case)

(value-keyword-case)